skopt/tree_opt.py

```diff
@@ -19,11 +20,23 @@
 from .learning import GradientBoostingQuantileRegressor
 from .learning import RandomForestRegressor
 from .space import Space
+from .utils import create_result
+from .utils import verbose_func
 
 
 def _tree_minimize(func, dimensions, base_estimator, n_calls,
                    n_points, n_random_starts, x0=None, y0=None,
-                   random_state=None, acq="EI", xi=0.01, kappa=1.96):
+                   random_state=None, acq="EI", xi=0.01, kappa=1.96,
+                   verbose=False, specs=None, callback=None):
```

Comment:
Is it legal for `specs` to ever not be passed as argument? I think it should be a mandatory argument.